tests/lib/sprintf/src/main.c

```diff
@@ -719,6 +729,170 @@ void test_sprintf_string(void)
 		     "sprintf(%%s) of REALLY_LONG_STRING doesn't match!\n");
 }
 
+/**
+ *
+ * @brief Test fprintf function
+ *
+ */
+
+void test_fprintf(void)
+{
+	int ret, i = 3;
+
+	ret = fprintf(stdout,  "%d", i);
+	zassert_equal(ret, 1, "fprintf failed!");
+
+	ret = printf("%d", 3);
+	zassert_equal(ret, 1, "printf failed!");
+}
+
+/**
+ *
+ * @brief Test vfprintf function
+ *
+ */
+
+int WriteFrmtd_vf(FILE *stream, char *format, ...)
+{
+	int ret;
+	va_list args;
+
+	va_start(args, format);
+	ret = vfprintf(stream, format, args);
+	va_end(args);
+
+	return ret;
+}
+
+void test_vfprintf(void)
+{
+	int ret;
+
+	ret = WriteFrmtd_vf(stdout,  "This %d", 3);
+	zassert_equal(ret, 6, "vfprintf This 3 failed");
+}
+
+/**
+ *
+ * @brief Test vprintf function
+ *
+ */
+
+int WriteFrmtd_v(char *format, ...)
+{
+	int ret;
+	va_list args;
+
+	va_start(args, format);
+	ret = vprintf(format, args);
+	va_end(args);
+
+	return ret;
+}
+
+void test_vprintf(void)
+{
+	int ret;
+
+	ret = WriteFrmtd_v("This %d", 3);
+	zassert_equal(ret, 6, "vprintf This 3 failed");
+}
+
+/**
+ *
+ * @brief Test fputs function
+ *
+ */
+
+void test_put(void)
+{
+	int ret;
+
+	ret = fputs("This 3", stdout);
+	zassert_equal(ret, 0, "fputs This 3 failed");
+
+	ret = fputs("This 3", stderr);
+	zassert_equal(ret, 0, "fputs This 3 failed");
+
+	ret = puts("This 3");
+	zassert_equal(ret, 0, "puts This 3 failed");
+
+	ret = fputc('T', stdout);
+	zassert_equal(ret, 84, "fputc T failed");
+
+	ret = fputc('T', stderr);
+	zassert_equal(ret, 84, "fputc T failed");
+
+	ret = fputc('T', stdin);
+	zassert_not_equal(ret, 84, "fputc T failed");
+}
+
+/**
+ *
+ * @brief Test fwrite function
+ *
+ */
+
+void test_fwrite(void)
+{
+	int ret;
+
+	ret = fwrite("This 3", 0, 0, stdout);
+	zassert_equal(ret, 0, "printf failed!");
```

Comment:
It should be "fwrite" rather than "printf".

Reply:
It seems to me that it has not been addressed yet.

Reply:
Done.